<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4VG4ENC" name="Home" tests="4" failures="2" timestamp="13 Jan 2023 07:46:09 GMT" time="106.316" errors="0">
  <testcase name="launchHomePage_Test" time="1.385" classname="com.automation.ui.digitaldeployer.testcases.index.IndexPageTest"/>
  <testcase name="verifyValidLogin" time="13.653" classname="com.automation.ui.digitaldeployer.testcases.login.LoginTest"/>
  <testcase name="launchAddNewWorkspacename_create_Test" time="21.417" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:93)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
  <testcase name="launchAddNewWorkspacename_create_Test" time="21.446" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:93)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
</testsuite> <!-- Home -->
